src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java

```diff
@@ -25,13 +31,58 @@ void getNoneAlcoholicRevenue() {
     }
 
     @Test
```

Comment:
also add short javadoc description to test methods
```suggestion
    /**
    * Test CashRegister.getLowAlcoholicRevenue method 
    */
    @Test
```